app/controllers/nwbib/Application.java

```diff
@@ -609,8 +609,8 @@ private static void uncache(List<String> ids) {
 			JsonNode json = pair.getLeft();
 			String label = pair.getRight();
 			int count = json.get("doc_count").asInt();
-			return (!label.contains("http")) && label.length() > String
```

Comment:
not sure, just guessing: wouldn't it be better to use `label.startWithIgnoreCase("http") ...?

Reply:
Yes, that would make more sense. I gave it a try but unfortunately, the labels at that point are actually HTML elements. Since this is unrelated to the fix in this issue, I'd prefer not to dig into that at this point.

Reply:
+1